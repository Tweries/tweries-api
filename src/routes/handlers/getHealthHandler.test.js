const getHealthHandler = require('./getHealthHandler');
const { mockSend, reqBase, resBase } = require('./mocks');

test('getHealthHandler', () => {
  const req = { ...reqBase };
  const res = { ...resBase };

  getHealthHandler(req, res);

  expect(mockSend.mock.calls[0][0]).toMatchSnapshot();
});
