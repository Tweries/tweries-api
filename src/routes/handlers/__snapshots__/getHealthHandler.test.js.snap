// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getHealthHandler 1`] = `
Object {
  "env": "test",
  "message": "ðŸ‘‹ hello, I'm healty",
  "name": "tweries-api",
  "version": "0.10.0",
}
`;
